crates/bevy_ecs/src/change_detection.rs

```diff
@@ -23,6 +23,17 @@ pub const CHECK_TICK_THRESHOLD: u32 = 518_400_000;
 /// Changes stop being detected once they become this old.
 pub const MAX_CHANGE_AGE: u32 = u32::MAX - (2 * CHECK_TICK_THRESHOLD - 1);
 
+pub trait ComponentMut<'a>: DetectChanges {
```

Comment:
Maybe a quick doc explaining that this trait is just to add the `const ENABLED: bool` to either `&mut T` or `Mut<T>` could be useful